packages/scripts/scripts/env/test-php.js

```diff
@@ -1,21 +1,39 @@
 /**
  * Node dependencies.
  */
-const { execSync } = require( 'child_process' );
+const path = require( 'path' );
 const { env } = require( 'process' );
 
+/**
+ * WordPress dependencies.
+ */
+const wpEnv = require( '@wordpress/env' );
+
 /**
  * Internal dependencies
  */
 const { getArgsFromCLI } = require( '../../utils' );
 
-const args = getArgsFromCLI();
+const args = getArgsFromCLI().join( ' ' );
 
-const localDir = env.LOCAL_DIR || 'src';
+// Default to CWD basename for the plugin to run.
+const pluginDir =
+	env.npm_package_wp_env_plugin_dir || path.basename( process.cwd() );
+const localDir = env.LOCAL_DIR || 'html';
 
-// Run PHPUnit with the working directory set correctly.
-execSync(
-	`npm run test:php -- -c /var/www/${ localDir }/wp-content/plugins/${ env.npm_package_wp_env_plugin_dir }/phpunit.xml.dist ` +
-		args.join( ' ' ),
-	{ cwd: env.WP_DEVELOP_DIR, stdio: 'inherit' }
-);
+// Executes phpunit in the Docker Service.
+wpEnv
+	.run( {
+		container: 'phpunit',
+		command: [
+			`phpunit -c /var/www/${ localDir }/wp-content/plugins/${ pluginDir }/phpunit.xml.dist ${ args }`,
```

Comment:
Would providing your own `-c` flag override the default (e.g. does last in get precedence?) Would be good if this could be easily overridden.

Reply:
Hm, that's a good idea. I'm not sure if it gets overridden. I'm not sure how to best expose this because it needs to know the path to the config file _inside_ docker, and then join that with the path to the config in the actual source code. 

So it's also currently bad because it won't work for wordpress or theme sources.

Reply:
I've added this to wp-env directly, so you could set the path via .wp-env.json. (it uses `$cwd/phpunit.dist.xml` by default if it exists)